README.md

```diff
@@ -94,19 +90,18 @@ I feel like learning about Wayland is a better use of my effort.
 Lack of EWMH means some applications (especially games) will break when 
 trying to go fullscreen.
 
-Authors
-=======
-
-See [the credits](CREDITS.md) for a full list of everybody who's helped.
-
-9wm is maintained by Jacob Adams <tookmund@gmail.com>.
 
-9wm was maintained by Neale Pickett <neale@woozle.org>.
+## Authors
 
-9wm was written by David Hogan (dhog@cs.su.oz.au), a postgraduate
-student at the Basser Department of Computer Science, University
-of Sydney <http://unauthorised.org/dhog/index.html>
-David Hogan died in 2003.
+See the [CREDITS.md](CREDITS.md) for a full list of everybody who's helped.
```

Comment:
Please revert. "See [the credits](CREDITS.md)" makes more sense than "See the [CREDITS.md](CREDITS.md)".

On a related note, your merge request should also include an edit to the credits adding yourself :)